lib/ProMotion/screen/screen_navigation.rb

```diff
@@ -75,6 +76,11 @@ def push_view_controller(vc, nav_controller=nil, animated=true)
       nav_controller.pushViewController(vc, animated: animated)
     end
 
+    def replace_nav_stack(screens, args = {})
+      args[:animated] ||= true
```

Comment:
This will always set `args[:animated]` to `true` (even if you set it manually to `false`).

Instead, try this:

```ruby
args[:animated] = true if args[:animated].nil?
```

Reply:
Oops, you're right.